### DISCUSSION TITLE
Is it possible to have two cloud mount points while transferring from one cloud to another?

### DISCUSSION THREAD

**Discussion by BEisem** [trapexit/mergerfs#1184] [Q&A]
I currently have mergerfs working to create a merged mount point for a single cloud mount and several local directories. I am considering moving from my current cloud provider to another, and was hoping to use mergerfs to make this process seamless and invisible to the local apps that are accessing the data.

So my thought was to add the second cloud provider as a mount point with the same directory structure, and add it to mergerfs. Then I can copy from one cloud provider to the other, and it would still look exactly the same to the apps. As the copies progress, I could just delete the files from the old cloud provider.

So first, would this work the way I'm thinking? 

And secondly, if one or more of the apps were to delete and/or upgrade files that exist on the cloud, would it still effectively delete files from the old cloud mount? My idea is that any new files, or upgrade files, would be placed in the new cloud mount, while deleting any old files from the old cloud mount.

Just wondering how this would work, or if it's even possible.

**Comment by trapexit** [comment]
1) Yes, it would work. With all the normal warnings regarding any caching you may have enabled. The fact it is a remote cloud mount doesn't change anything to mergerfs.
2) It depends entirely on your setup. Again, nothing is special about it being a cloud mount. If a file is removed it will call the unlink policy and act accordingly.
